### ISSUE TITLE
Bug in LoadBalancedEc2Service -- must have at least one of 'memoryLimitMiB' or 'memoryReservationMiB' specified

### ISSUE THREAD

**Issue by Fejuto** [awslabs/aws-cdk#2263]
memoryLimitMiB is not optional at the moment while it should be; validate doesn't check for memoryReservationMiB.
```
validate() {
        const ret = super.validate();
        if (util_1.isEc2Compatible(this.compatibility)) {
            // EC2 mode validations
            // Container sizes
            for (const container of this.containers) {
                if (!container.memoryLimitSpecified) {
                    ret.push(`ECS Container ${container.node.id} must have at least one of 'memoryLimitMiB' or 'memoryReservationMiB' specified`);
                }
            }
        }
        return ret;
    }
```
 - CDK Version 0.28

**Comment by SoManyHs** [comment]
Hi @Fejuto,

Could you include the example you are trying to run? This validation exists in the [task-definition.ts](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs/lib/base/task-definition.ts#L294-L296)construct, but just want to be clear on your use case. Thanks!

**Comment by Fejuto** [comment]
```

**Comment by SoManyHs** [comment]
Hi @Fejuto, sorry for the delay in response. This should be fixed in #2463. Thanks for your patience!
